src/audio/drc/drc_math_generic.c

```diff
@@ -207,14 +211,15 @@ inline int32_t drc_inv_fixed(int32_t x, int precision_x, int precision_y)
 	const int32_t A1 = Q_CONVERT_FLOAT(-21.25031280517578125f, qc);
 	const int32_t A0 = Q_CONVERT_FLOAT(7.152250766754150390625f, qc);
 	int e;
```

Comment:
Change int e to int_32_t e

Reply:
ack, if these should be the same types to make the algo portable to non xtensa arch.

Reply:
@johnylin76 this change is still missing